.gitmodules

```diff
@@ -14,6 +14,3 @@
 [submodule "web/k-web-theme"]
 	path = web/k-web-theme
 	url = https://github.com/runtimeverification/k-web-theme
-[submodule "tests/foundry/lib/forge-std"]
```

Comment:
You might also need to remove the `tests/foundry/lib/forge-std` submodule directly from the repo.